### PR TITLE
Add Webchain support #2

### PR DESCRIPTION
**Pull request by lukaszmatczak**
Please add Webchain support. We changed chain ID already so there's no conflict anymore.

```bitcointalk ANN    : https://bitcointalk.org/index.php?topic=3649170.0
homepage           : https://webchain.network/
block explorer     : https://explorer.webchain.network/
slip0044 index     : 227
chainId            : 24484
```

### CODE REVIEW

**Comment by nickmccurdy** [comment]
Please update the failing snapshot test: https://travis-ci.org/MyCryptoHQ/MyCrypto/jobs/517760039

**Comment by lukaszmatczak** [comment]
@nickmccurdy Done

File: common/features/config/networks/static/reducer.ts [hunk 1]
```diff
@@ -850,6 +851,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+    isCustom: false,
+    color: '#0092ee',
+    blockExplorer: makeExplorer({
+      name: 'Webchain Explorer',
+      origin: 'https://explorer.webchain.network'
+    }),
+    tokens: [],
+    contracts: [],
+    dPathFormats: {
+      [SecureWalletName.TREZOR]: WEB_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: Please include links to the relevant pull request in third party repositories. It saves time for those reviewing the PR. 

No pull request currently exists for Trezor support. Please remove TREZOR support from this PR until after you can demonstrate Ledger hardware wallet support.

File: common/features/config/networks/static/reducer.ts [hunk 2]
```diff
@@ -850,6 +851,31 @@ export const STATIC_NETWORKS_INITIAL_STATE: types.ConfigStaticNetworksState = {
...
+      [SecureWalletName.SAFE_T]: WEB_DEFAULT,
+      [SecureWalletName.LEDGER_NANO_S]: WEB_DEFAULT,
```
Comment (mkrufky) [inline common/features/config/networks/static/reducer.ts]: As I mentioned above, please include links to the relevant pull request in third party repositories. It saves time for those reviewing the PR. 

No pull request currently exists for Ledger support. Please remove LEDGER_NANO_S support from this PR until after you can demonstrate Ledger hardware wallet support.

Lucas, you may have heard that I got the Ledger working for WebChain, but we can't include support for it in MyCrypto (nor MEW) until after the pull request has been issued upstream to LedgerHQ.  We still have some work to do before that PR can be made, so LEDGER_NANO_S support should be left out of this until then.

**Comment by SharonManrique** [comment]
Merged into https://github.com/MyCryptoHQ/MyCrypto/pull/2479

**Pull request closed without merging**